app/views/pages/new.html.erb

```diff
@@ -1,20 +1,20 @@
 <% set_page_title(title_with_error_prefix(t("page_titles.new_page"), @question_input.errors.any?)) %>
-<% content_for :back_link, govuk_back_link_to(form_pages_path(current_form)) %>
+<% content_for :back_link, govuk_back_link_to(form_pages_path(current_form.id)) %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l">
-      <span class="govuk-caption-l"><%= t("pages.question") %> <%= current_form.page_number(@page) %></span>
+      <span class="govuk-caption-l"><%= t("pages.question") %> <%= current_form.page_number(nil) %></span>
       <span class="govuk-visually-hidden"> - </span>
       <%= t("pages.heading") %>
     </h1>
 
     <%= render partial: 'form', locals: { is_new_page: true,
                                           form_object: current_form,
-                                          page_object: @page,
+                                          page_object: nil,
```

Comment:
I may be misunderstanding how this works, but do we need to pass in the page_object at all here? removing it entirely the tests still pass

Reply:
Oh, you're right! I was thinking of these as function arguments and so needing to be defined, but they're not, so it's fine... I'll update the commit to remove that line.